code-server/main.tf

```diff
@@ -95,6 +95,12 @@ variable "use_cached" {
   default     = false
 }
 
+variable "extensions_dir" {
+  type        = string
+  description = "Override the directory to store extensions in. By default extensions are in ~/.local/share/code-server/extensions."
```

Comment:
Should we say `$XDG_DATA_HOME/code-server/extensions`?  That will probably almost always be `~/.local/share` but maybe in some cases it will be configured differently.

Reply:
True, i would also mention that the following code does not support `XDG_DATA_HOME`

```bash
if [ ! -f ~/.local/share/code-server/User/settings.json ]; then
  echo "⚙️ Creating settings file..."
  mkdir -p ~/.local/share/code-server/User
  echo "${SETTINGS}" > ~/.local/share/code-server/User/settings.json
fi
```

Reply:
Oh right good catch.  Maybe we should just set `default` to `~/.local/share/code-server/extensions`?

Reply:
We could, it _might_ be a breaking change for some, but making the default value `~/.local/share/code-server/extensions` is much more explicit.

Reply:
Good point.  Maybe we just update the description here for this PR and we can try figuring out what to do with this later.